workshop/content/30-python/70-advanced-topics/200-pipelines/4000-build-stage.md

```diff
@@ -12,14 +12,14 @@ Create a new file in `cdk_workshop` called `pipeline_stage.py` with the code bel
 from aws_cdk import (
     core
 )
-from pypipworkshop_stack import PypipworkshopStack
+from cdk_workshop_stack import CdkWorkshopStack
```

Comment:
```suggestion
from .cdk_workshop_stack import CdkWorkshopStack
```

Reply:
thanks, I added the fix here [b3d4f07](https://github.com/aws-samples/aws-cdk-intro-workshop/pull/320/commits/b3d4f073ff6fa29e8da4f583306ee65c6ccae9bf)